### DISCUSSION TITLE
Anyone using Fluent Assertions in Unity 2022.3 beside me?

### DISCUSSION THREAD

**Discussion by MadInventorQDB** [fluentassertions/fluentassertions#2688] [General]
I have a background in application and web services development and have experimented with Unity for years. Recently, I decided to get serious about applying my engineering knowledge, focusing on unit testing, and chose to use Fluent Assertions for its superior features. I'm curious about how others are integrating this.

I pulled the code, built the .NET Framework 4.7 version, and confirmed it works with Unity 2022.3. However, I'm wondering if there's a better method. Using a package seems more efficient than relying on a locally built DLL, but I'm unsure how to connect the Unity Package Manager to the repository. I've attempted to use https://github.com/fluentassertions/fluentassertions.git without success. Any insights would be appreciated. :innocent:

**Comment by dennisdoomen** [comment]
Why don't you use the NuGet package?

**Reply by MadInventorQDB** [reply]
Thanks for the suggestion.

I didn't think there was a Unity NuGet Package Manager, but since you suggested it, I checked. I was wrong. There is one.
https://github.com/GlitchEnzo/NuGetForUnity

However, NuGetForUnity doesn't seem to be able to install Fluent Assertions. The install is disabled and no version is shown. The next package install is enabled and has a version. 

![image](https://github.com/fluentassertions/fluentassertions/assets/121001760/76c4a285-942e-4ca8-bf7e-8c90d024370d)

But, because of your suggestion I might pull Microsoft.Extensions.DependencyInjection or Autofac to sprinkle some DI magic on my project.

Thanks again.
